kratos/solving_strategies/schemes/scheme.h

```diff
@@ -393,17 +393,17 @@ class Scheme
         KRATOS_TRY
         //initialize solution step for all of the elements
         ElementsArrayType& pElements = r_model_part.Elements();
-        ProcessInfo& CurrentProcessInfo = r_model_part.GetProcessInfo();
```

Comment:
i am not in favor of changing the base scheme.h without a good reason. Also i would say that the current naming is correct

Reply:
There was a warning of unused

Reply:
Cammelcase

Reply:
It was camel case before, so to my understading it was correct. In any case i would not change this file only for aesthetic reasons

Reply:
That's why I corrected again, a mistake, sorry